pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAgents.scala

```diff
@@ -89,7 +89,7 @@ trait SierraAgents {
 
   def getLabel(subfields: List[MarcSubfield]): Option[String] =
     subfields.filter { s =>
-      List("a", "b", "c", "d", "t").contains(s.tag)
+      List("a", "b", "c", "t").contains(s.tag)
```

Comment:
I don't think we should remove this from the label as it will land up with us having screwy punctuation on the frontend again.

Ping @jtweed @jennpb

Reply:
```suggestion
      List("a", "b", "c", "d", "t").contains(s.tag)
```